### PR TITLE
Contributing guide

### PR DESCRIPTION
**Pull request by walaszczykm**
### Type of change

- [x] Docs
- [ ] Bug fix
- [ ] Feature

### Packages

- [x] @livechat/widget-core
- [x] @livechat/widget-react
- [x] @livechat/widget-vue
- [x] @livechat/widget-angular

### Issue

N/A

### Description

- Create a dedicated `CONTRIBUTING.md` file with contributing guide extracted from the main `README.md` and extended with a description of newly added packages unit tests as also issue reporting.
- Extend and reorganize already existing sections.
- Fix duplicated property description in packages `README.md` files
- Update packages `README.md` files with contributing guide link

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+We track public bugs in [GitHub Issues](https://github.com/livechat/chat-widget-adapters/issues). If you would like to report one we recommend filling the [Bug report](https://github.com/livechat/chat-widget-adapters/issues/new/choose).
+
+## Development Workflow
+
+- Install dependecies using `npm install`. This will also bootstrap a monorepo installing dependencies inside each package and linking them together.
+
+- Build libraries using `npm run build` at the top level or inside single package.
+
+- Start watch mode for all adapters packages running `npm start packages`.
```
Comment (BartoszGamza) [inline CONTRIBUTING.md:17]:
```suggestion
- Start watch mode for all the adapters packages running `npm start packages`.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+- Start example apps showcasing usage of componenets for each framework using `npm start examples`.
```
Comment (BartoszGamza) [inline CONTRIBUTING.md:19]:
```suggestion
- Start example apps showcasing the usage of components for each framework using `npm start examples`.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+- Start watch mode for given adapter with its example app running `npm start vue/react/angular`.
```
Comment (BartoszGamza) [inline CONTRIBUTING.md:21]:
```suggestion
- Start watch mode for the given adapter with its example app running `npm start vue/react/angular`.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+## Versioning
+
+This project uses [lerna](https://lerna.js.org/) to manage versions and publishing. It uses locked [semantic versioning](https://semver.org/) so each package has the same version at the time. You don't need and shouldn't manullay update touched packages versions. Changes will be published as soon as the next release will be rolled out. In order to properly update all packages to a new version checkout to `master` branch and run `npx lerna version`. It will ask about update type and if accepted update all packages and push tagged commit with changes in `package*.json` files.
+
+## Comitting
+
+The project is setup with a code quality tools like [`prettier`](https://npm.im/prettier), [`eslint`](https://npm.im/eslint) and [`manypkg`](https://npm.im/@manypkg/cli). They are being automatically run against changed files on `pre-commit` hook.
```
Comment (BartoszGamza) [inline CONTRIBUTING.md:29]:
```suggestion
The project is set up with a code quality tools like [`prettier`](https://npm.im/prettier), [`eslint`](https://npm.im/eslint) and [`manypkg`](https://npm.im/@manypkg/cli). They are being automatically run against changed files on `pre-commit` hook.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+|    └─── angular          // example Angular app
+|
+└─── scripts               // CLI commands used in root npm scripts
+```
+
+## Testing
+
+### Manual testing
+
+In order to manually test introduced changes, use one of example apps for the framework whcih package you are chaning. It contains ready to use setup for testing basic functionalities. If you have added something new, make sure to extend the example with possiblity to visually test your changes.
```
Comment (BartoszGamza) [inline CONTRIBUTING.md:66]:
```suggestion
In order to manually test introduced changes, use one of example apps for the framework whcih package you are changing. It contains ready to use setup for testing basic functionalities. If you have added something new, please make sure to extend the example with possibility to visually test your changes.
```

**Review by BartoszGamza — COMMENTED** [review]
Left some grammar/stylistic comments, great description!

**Review by BartoszGamza — APPROVED** [review]
🚀

**Review by klarzynskik — APPROVED** [review]
Neat! 🔝

**Pull request merged**